MMM-MBTA.js

```diff
@@ -613,6 +631,34 @@ Module.register("MMM-MBTA", {
         // Sorts them according to ETA time
         this.stationData.sort((a,b) => (a.preETA - b.preETA));
 
+        if (this.config.flipDirection && this.config.direction.length > 0)
+        {
+            //after flipTime, invert direction 
+             var flipTime = moment().toDate();
+             flipTime.setHours(this.config.flipHour);
+             flipTime.setMinutes(0);
+
+            if (moment().isSameOrAfter(flipTime)) {
+
+                    if (!this.directionFlipped)
+                    {
+                        this.filterDirection[0] = (1 - this.filterDirection[0]).toString();
+                        this.directionFlipped = true;
+                    }
+            }
+            else
+            {
+                this.directionFlipped = false;
+            }
+
+        }
+
+        //  Applys directional filter
+        if (this.filterDirection.length > 0)
+        {
+            this.stationData = this.stationData.filter(obj => (obj.directionId == this.filterDirection));
```

Comment:
Double equals is discouraged. This should be `===`.
```suggestion
            this.stationData = this.stationData.filter((obj) => obj.directionId === this.filterDirection);
```

Reply:
ended up it should have been =